ovirtlib4/vms.py

```diff
@@ -68,6 +68,10 @@ def backups(self):
     def reported_devices(self):
         return VmReportedDevices(connection=self.service)
 
+    @property
+    def snapshots(self):
```

Comment:
maybe it will be better to keep the same obj name
```suggestion
    def vm_snapshots(self):
```

Reply:
It is a class method that belong to a VM class, so it is not acceptable to duplicate the class name into the method

e.g:
the class is a VM instance, so you will have 
vm = VM()
vm.vm_snapshots()
vm.vm_disks()
vm.vm_nics()

instead of:
vm.snapshots()
vm.disks()
vm.nics()

Reply:
OK, Thanks!!